tests/tool/Integration.Tests/Integration.Tests.csproj

```diff
@@ -12,6 +12,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <Content Remove="IntegrationScenarios\MauiSample\**" />
```

Comment:
I don't see any files matching this pattern. Are these coming later?

Reply:
woops, yes removing this lol. these are coming later!